src/ProcessManager/ChromeManager.php

```diff
@@ -91,6 +91,12 @@ private function getDefaultArguments(): array
             $args[] = '--no-sandbox';
         }
 
+        // Add custom arguments with PANTHER_CHROME_ARGUMENTS
+        if ($_SERVER['PANTHER_CHROME_ARGUMENTS'] ?? false) {
+            $arguments = \explode(' ', $_SERVER['PANTHER_CHROME_ARGUMENTS']);
+            $args = \array_merge($args, $arguments);
```

Comment:
**array_merge** and **explode** are not in the list of functions replaced by opcode. The backslash is not necessary .

Reply:
Oh ! I agree with you, we can see all functions covered by opcode [here](https://github.com/php/php-src/blob/f2db305fa4e9bd7d04d567822687ec714aedcdb5/Zend/zend_compile.c#L3872). PHPStan report that and I think it's not false, because there are two reasons, opcode and also global namespace. I think we can keep this leadslash. Maybe in a future, this functions will be covered by opcode and it is not necessary to configure PHPStan to report only this functions :)

Reply:
Adding more functions as opcodes would make the engine slower, because routing more opcodes requires more CPU. So this will not happen :)
The policy is to never add `\`, except when it provides a significant advantage, i.e. for those special functions that php-cs-fixer/fabbot handle.

Reply:
Ok but we can see on Panther other array_merge with leadslash. Even we change that, I think new PR is necessary because it's an other issue.

Reply:
It would be nice to have a new fixer to remove the leading blackslash when it's useless.

Reply:
deserves an issue on php-cs-fixer :)

Reply:
@matthieumota can you remove these two backslashes so we can merge this one?

Reply:
@dunglas Done !